src/test/java/com/taskviewer/api/postgres/PgTasksUnitTest.java

```diff
@@ -90,7 +91,6 @@ void verifiesByUsername() {
 		List<Task> tasks = new ArrayList<>(2);
 		tasks.add(task1);
 		tasks.add(task2);
-		String username = "testUsername";
 		when( jdbc.query(PgTasks.FIND_BY_USERNAME, view, username)).thenReturn(tasks);
 
```

Comment:
we don't tolerate empty lines
read: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html